<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FDs</class>
 <widget class="QWidget" name="FDs">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сеансы ДС</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/table.png</normaloff>:/res/1/table.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Обновить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1328013711_Refresh.ico</normaloff>:/res/1328013711_Refresh.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnList">
        <property name="text">
         <string>Просмотр</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Text.png</normaloff>:/res/Text.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVigr">
        <property name="text">
         <string>Выгрузить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/folder_go.png</normaloff>:/res/1/folder_go.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/bin.png</normaloff>:/res/1/bin.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDateEdit" name="dtn">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dtk">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDat">
        <property name="text">
         <string>Отбор по датам</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFiltr">
        <property name="text">
         <string>Фильтр НПН</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/filter.png</normaloff>
          <normalon>:/res/1/filter_clear.png</normalon>
          <disabledoff>:/res/1/filter.png</disabledoff>
          <disabledon>:/res/1/filter_clear.png</disabledon>
          <activeoff>:/res/1/filter.png</activeoff>
          <activeon>:/res/1/filter_clear.png</activeon>
          <selectedoff>:/res/1/filter.png</selectedoff>
          <selectedon>:/res/1/filter_clear.png</selectedon>:/res/1/filter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbNpn"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>799</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
